beacon-chain/p2p/fork.go

```diff
@@ -86,10 +89,25 @@ func addForkEntry(
 	if err != nil {
 		return nil, err
 	}
+	currentSlot := helpers.SlotsSince(genesisTime)
+	currentEpoch := helpers.SlotToEpoch(currentSlot)
+	if roughtime.Now().Before(genesisTime) {
+		currentSlot, currentEpoch = 0, 0
+	}
+	fork, err := p2putils.Fork(currentEpoch)
+	if err != nil {
+		return nil, err
+	}
+
 	nextForkEpoch := params.BeaconConfig().NextForkEpoch
+	nextForkVersion := params.BeaconConfig().NextForkVersion
```

Comment:
Should we add a unit test for this? This looks isolated enough from the p2p package and should be a good unit test candidate

Reply:
ok done, added it in